app/javascript/components/Modal.vue

```diff
@@ -4,21 +4,21 @@
       <div class="modal__content">
         <button class="modal__close" @click="closeModal()">X</button>
 
-        <p><strong>ID</strong> {{ modalContent.id }}</p>
-        <p><strong>Project Title</strong> {{ modalContent.projectTitle }}</p>
-        <p><strong>Donor name</strong> {{ modalContent.donor }}</p>
+        <p><strong>ID</strong> {{ modalContent.number }}</p>
```

Comment:
Since we are going to remove `number` in the near future, you can still use `id` here.